### PR TITLE
Update to latest versions of all the things

### PR DESCRIPTION
**Pull request by jpstroop**

### CODE REVIEW

**Comment by jpstroop** [comment]
OK; that wasn't tedious at all! I think this is ready for review.

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ script: rake ci
 # environment
 language: ruby
 rvm:
-  - "2.2"
+  - "2.2.3"
```
Comment (jpstroop) [inline .travis.yml]: @eefahy this is important--Jekyll has a dependency on `ActiveSomethingOrOther` that only works with Ruby >2.2.3. I think that's what's on the server, but FYI.

**Comment by zimeon** [comment]
Are we ready to accept breaking every section link that is out there through the new-style prefix of section numbers? Is there really no way to avoid this?

File: .travis.yml (hunk 1 above)
Comment (eefahy) [inline .travis.yml]: Yep that's on the server:
``` console
▶ ssh iiif          
Last login: Wed Mar 29 20:49:07 2017 from c-67-188-100-140.hsd1.ca.comcast.net
[iiif@ip-172-31-44-83 ~]$ rvm list rubies

rvm rubies

=* ruby-2.2.3 [ x86_64 ]

# => - current
# =* - current && default
#  * - default
```

**Comment by jpstroop** [comment]
Closing. Will send a new PR that pins Jekyll

**Comment by jpstroop** [comment]
See https://github.com/gettalong/kramdown/issues/424

**Pull request closed without merging**